src/System.Private.ServiceModel/tests/Common/Infrastructure/src/testproperties.props

```diff
@@ -86,26 +86,39 @@
       <NegotiateTestUpn>testuser@DOMAIN.CONTOSO.COM</NegotiateTestUpn>
     </PropertyGroup>
 
+    <!-- Convention for TestProperties values to use by [ConditionalFact]:
+      'true' = yes, the condition is satisfied
+      'false' = no, the condition is not satisfied
+      empty string = self-detect at runtime if possible, else assume 'false'
+    -->
     <PropertyGroup Condition="'$(Domain_Joined)' == ''">
-      <Domain_Joined>false</Domain_Joined>
+      <Domain_Joined></Domain_Joined>
```

Comment:
TestProperties.props is not the place to look for the complete list of TestProperties. It is the place to provide default value if a TestProperty has one. TestProperties.targets is the place to look for complete list. If our build does not require every TestProperty to show up in TestProperties.props, sooner or later people will need to look TestProperties.targets for the ultimate truth anyway.